Warning: ‘void QTime::start()’ is deprecated

```shell
/home/USER/Development/OpenMs/src/tests/class_tests/openms_gui/source/GUI/TOPPView_test.cpp:75:10: warning: ‘void QTime::start()’ is deprecated: Use QElapsedTimer instead [-Wdeprecated-declarations]
   75 |   t.start();
      |   ~~~~~~~^~
In file included from /usr/include/qt/QtCore/qcborvalue.h:44,
                 from /usr/include/qt/QtCore/qcborarray.h:43,
                 from /usr/include/qt/QtCore/QtCore:38,
                 from /usr/include/qt/QtTest/QtTestDepends:3,
                 from /usr/include/qt/QtTest/QtTest:3,
                 from /home/USER/Development/OpenMs/src/tests/class_tests/openms_gui/include/GUI/TOPPView_test.h:39,
                 from /home/USER/Development/OpenMs/src/tests/class_tests/openms_gui/source/GUI/TOPPView_test.cpp:35:
/usr/include/qt/QtCore/qdatetime.h:235:55: note: declared here
  235 |     QT_DEPRECATED_X("Use QElapsedTimer instead") void start();
      |                                                       ^~~~~
/home/USER/Development/OpenMs/src/tests/class_tests/openms_gui/source/GUI/TOPPView_test.cpp:76:65: warning: ‘int QTime::elapsed() const’ is deprecated: Use QElapsedTimer instead [-Wdeprecated-declarations]
   76 |   while (!modal_key_sequence_.isEmpty () && max_wait > t.elapsed())
      |                                                        ~~~~~~~~~^~

```

```shell
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/Plot1DCanvas.cpp: In member function ‘void OpenMS::Plot1DCanvas::paint(QPainter*, QPaintEvent*)’:
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/Plot1DCanvas.cpp:720:16: warning: ‘void QTime::start()’ is deprecated: Use QElapsedTimer instead [-Wdeprecated-declarations]
  720 |     timer.start();
      |     ~~~~~~~~~~~^~
In file included from /usr/include/qt/QtCore/QDate:1,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/DATASTRUCTURES/DateTime.h:40,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/METADATA/DataProcessing.h:39,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/METADATA/ChromatogramSettings.h:43,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/KERNEL/MSChromatogram.h:38,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/KERNEL/MSExperiment.h:40,
                 from /home/USER/Development/OpenMs/src/openms_gui/include/OpenMS/VISUAL/LayerDataBase.h:42,
                 from /home/USER/Development/OpenMs/src/openms_gui/include/OpenMS/VISUAL/PlotCanvas.h:44,
                 from /home/USER/Development/OpenMs/src/openms_gui/include/OpenMS/VISUAL/Plot1DCanvas.h:41,
                 from /home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/Plot1DCanvas.cpp:35:
/usr/include/qt/QtCore/qdatetime.h:235:55: note: declared here
  235 |     QT_DEPRECATED_X("Use QElapsedTimer instead") void start();
      |                                                       ^~~~~
```
One more file

```shell
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/Plot2DCanvas.cpp: In member function ‘virtual void OpenMS::Plot2DCanvas::paintEvent(QPaintEvent*)’:
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/Plot2DCanvas.cpp:1454:26: warning: ‘void QTime::start()’ is deprecated: Use QElapsedTimer instead [-Wdeprecated-declarations]
 1454 |       overall_timer.start();
      |       ~~~~~~~~~~~~~~~~~~~^~
In file included from /usr/include/qt/QtCore/QDate:1,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/DATASTRUCTURES/DateTime.h:40,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/METADATA/DataProcessing.h:39,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/METADATA/ID/ProcessingStep.h:37,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/METADATA/ID/IdentificationData.h:37,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/KERNEL/BaseFeature.h:39,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/KERNEL/ConsensusFeature.h:38,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/KERNEL/ConsensusMap.h:40,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/FORMAT/ConsensusXMLFile.h:39,
                 from /home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/Plot2DCanvas.cpp:36:
/usr/include/qt/QtCore/qdatetime.h:235:55: note: declared here
  235 |     QT_DEPRECATED_X("Use QElapsedTimer instead") void start();
      |                                                       ^~~~~
```

--- a/src/openms_gui/include/OpenMS/VISUAL/APPLICATIONS/TOPPViewBase.h
+++ b/src/openms_gui/include/OpenMS/VISUAL/APPLICATIONS/TOPPViewBase.h
@@ -60,6 +60,7 @@
 #include <QtWidgets/QActionGroup>
 #include <QtCore/QStringList>
 #include <QtCore/QProcess>
+#include <QElapsedTimer>
 
 class QAction;
 class QComboBox;
@@ -523,7 +524,7 @@ protected:
       UInt window_id;
       Size spectrum_id;
       QProcess* process = nullptr;
-      QTime timer;
+      QElapsedTimer timer;
       bool visible_area_only;
     } topp_;
     //@}

--- a/src/openms_gui/source/VISUAL/Plot1DCanvas.cpp
+++ b/src/openms_gui/source/VISUAL/Plot1DCanvas.cpp
@@ -56,7 +56,7 @@
 // Qt
 #include <QMouseEvent>
 #include <QPainter>
-#include <QtCore/QTime>
+#include <QElapsedTimer>
 #include <QtWidgets/QFileDialog>
 #include <QtWidgets/QInputDialog>
 #include <QtWidgets/QMenu>
@@ -716,7 +716,7 @@ namespace OpenMS
 
   void Plot1DCanvas::paint(QPainter* painter, QPaintEvent* e)
   {
-    QTime timer;
+    QElapsedTimer timer;
     timer.start();
 
     // clear

--- a/src/openms_gui/source/VISUAL/Plot2DCanvas.cpp
+++ b/src/openms_gui/source/VISUAL/Plot2DCanvas.cpp
@@ -57,7 +57,7 @@
 #include <QMouseEvent>
 #include <QPainter>
 #include <QPolygon>
-#include <QtCore/QTime>
+#include <QElapsedTimer>
 #include <QtWidgets/QComboBox>
 #include <QtWidgets/QMenu>
 #include <QtWidgets/QMessageBox>
@@ -1448,7 +1448,7 @@ namespace OpenMS
 #endif
 
     //timing
-    QTime overall_timer;
+    QElapsedTimer overall_timer;
     if (show_timing_)
     {
       overall_timer.start();
@@ -1472,7 +1472,7 @@ namespace OpenMS
 
       buffer_.fill(QColor(String(param_.getValue("background_color").toString()).toQString()).rgb());
       painter.begin(&buffer_);
-      QTime layer_timer;
+      QElapsedTimer layer_timer;
 
       for (Size i = 0; i < getLayerCount(); i++)
       {

--- a/src/tests/class_tests/openms_gui/source/GUI/TOPPView_test.cpp
+++ b/src/tests/class_tests/openms_gui/source/GUI/TOPPView_test.cpp
@@ -35,7 +35,7 @@
 #include <TOPPView_test.h>
 
 #include <QTimer>
-
+#include <QElapsedTimer>
 #include <OpenMS/VISUAL/APPLICATIONS/TOPPViewBase.h>
 #include <OpenMS/SYSTEM/File.h>
 #include <OpenMS/VISUAL/EnhancedTabBar.h>
@@ -71,7 +71,7 @@ void TestTOPPView::waitForModalWidget(const int max_wait, const String& line)
   }
 
 
-  QTime t;
+  QElapsedTimer t;
   t.start();
   while (!modal_key_sequence_.isEmpty () && max_wait > t.elapsed())
   {

